Sources/RswiftCore/RswiftCore.swift

```diff
@@ -40,6 +36,14 @@ public struct RswiftCore {
         StringsStructGenerator(localizableStrings: resources.localizableStrings),
       ]
 
+      do {
+        let colorPaletteGenerator = ColorPaletteStructGenerator(palettes: resources.colors)
+        let colorPaletteGeneratorStruct = colorPaletteGenerator.generatedStructs(at: callInformation.accessLevel, prefix: "")
+        if !colorPaletteGeneratorStruct.externalStruct.isEmpty {
+          generators.append(colorPaletteGenerator)
+        }
+      }
```

Comment:
Maybe this block should have a comment that this generator is deprecated and should be removed in R.swift 5?